src/shared/_helpers/ArrayHelpers.ts

```diff
@@ -0,0 +1,33 @@
+class ArrayHelpers {
+  /**
+   * Get a random item in the array.
+   *
+   * @param {any[]} array Array
+   * @return {boolean}
+   */
+  static getRandomItem(array: any[]): any {
+    return array[Math.floor(Math.random() * array.length)];
+  }
+
+  /**
+   * Check if a value is an array.
+   *
+   * @param {any} value Value to check
+   * @return {boolean}
+   */
+  static isArray(value: any): any {
+    return Array.isArray(value);
+  }
+
+  /**
+   * Check if an array has at least one item.
+   *
+   * @param {any} value Value to check
+   * @return {boolean}
+   */
+  static isEmpty(value: any): any {
```

Comment:
Can you fix the any types please ?

```suggestion
   * @param {unknown} value Value to check
   * @return {boolean}
   */
  static isEmpty(value: unknown): boolean {
```

Reply:
Le type `unknown` ne fonctionnera pas dans ce cas là car on y applique une méthode par la suite `value.length === 0` (_'value' is of type 'unknown'_)